Serverless-Functions/homework/responses/1.5-blog.md

```diff
@@ -0,0 +1,60 @@
+**Let’s wrap up what you learned and did in Week 1: downloading and using Visual Studio Code, learning to navigate GitHub and committing changes, and deploying a simple Azure function.**
```

Comment:
try taking a look at the week 2 or 3 blog for a "template" on how to format everything :)